bld/namelist_files/namelist_defaults_pop.xml

```diff
@@ -1378,10 +1378,12 @@
 <iron_flux_input%scale_factor ocn_grid="gx3v7">1.79e6</iron_flux_input%scale_factor>
 <iron_flux_input%scale_factor ocn_grid="gx1v6">1.79e6</iron_flux_input%scale_factor>
 <iron_flux_input%scale_factor ocn_grid="gx1v7">1.79e6</iron_flux_input%scale_factor>
+<iron_flux_input%scale_factor ocn_grid="tx0.1v3">1.79e6</iron_flux_input%scale_factor>
 
 <fesedflux_input%filename ocn_grid="gx3v7">ocn/pop/gx3v7/forcing/fesedflux_gx3v7_cesm1_97_2017.nc</fesedflux_input%filename>
 <fesedflux_input%filename ocn_grid="gx1v6">ocn/pop/gx1v6/forcing/fesedfluxTot_gx1v6_cesm2_2018_c180618.nc</fesedflux_input%filename>
 <fesedflux_input%filename ocn_grid="gx1v7">ocn/pop/gx1v6/forcing/fesedfluxTot_gx1v6_cesm2_2018_c180618.nc</fesedflux_input%filename>
+<fesedflux_input%filename ocn_grid="tx0.1v3">ocn/pop/tx0.1v3/tmp/fesedflux_total_reduce_oxic_POP_tx0.1v3.c200420.nc</fesedflux_input%filename>
```

Comment:
Need to either move this to proper directory (and import it to inputdata repo) or generate updated version